app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -64,6 +64,26 @@ class DradisDatatable {
       },
       lengthChange: false,
       pageLength: 25,
+      stateSave: true,
+      stateSaveCallback: function(settings, data) {
+        var dataTableDomId = settings.oInstance[0].id;
+        var localStorageKey = `Datatables_${dataTableDomId}_${window.location.pathname}`;
+
+        // Check if the table has been loaded before so that we can skip
+        // the manually hidden columns on page load.
+        var initialLoad = false;
```

Comment:
Do we really need this boolean? Can we just check if there is an "initial load" if the localstorage is empty or not? We access the localstorage in the `hideColumns` method anyway.

Reply:
Moved to the class's attribute.